src/gui/dialog/LatexDialog.cpp

```diff
@@ -97,6 +98,14 @@ void LatexDialog::load()
 {
 	XOJ_CHECK_TYPE(LatexDialog);
 	gtk_text_buffer_set_text(this->textBuffer, this->theLatex.c_str(), -1);
+
+	// preselect all text in the box if desired
+	if (this->preselect) {
```

Comment:
This flag is unecessary as the method `load()` is only called by the method `show()` which is called on dialog creation.